### PR TITLE
Fixed running repocop locally and improved logic on what to run in which setting

### PR DESCRIPTION
**Pull request by JuliaBrigitte**
## What does this change?
We are able to run repocop locally again
We changed the logic and what to run in which environment to make it easier to understand.
PROD talks to branch_protector and the anghammarad
CODE talks to branch_protector
DEV does not talk to anything by default, but you can set ENABLE_MESSAGING=true for testing purposes

## Why?
To be able to test locally

## How has it been verified?
It worked locally.

### CODE REVIEW

File: packages/cloudquery/README.md [hunk 1]
```diff
@@ -52,6 +53,8 @@ It includes:
 > **Note**
 > You can also use other Postgres clients, such as `psql` to query the data, or even your IDE!
 
+To configure your db connection, go the RDS in the deploytool account and get the DB Identifier for code or prod, as 
+authentification for the postgres db you can use SecretsmanagerAuth with the arn for the secret.
```
Comment (akash1810) [inline packages/cloudquery/README.md:57]: This doesn't feel related to this change. Is it needed? I'd advocate against connecting to RDS locally too, so unsure when this advice would be needed/followed?

File: .env [hunk 2]
```diff
@@ -32,3 +32,5 @@ LOCAL_DB_USER=postgres
 LOCAL_DB_PASSWORD=not_at_all_secret
 LOCAL_DB_HOSTNAME=localhost
 LOCAL_DB_PORT=5432
+
+ENABLE_MESSAGING=false
```
Comment (akash1810) [inline .env:36]: Worth adding an inline comment here to explain what this flag controls?

**Review by akash1810 — APPROVED** [review]
LGTM.

Let's deploy to CODE before merging to ensure the changes to the AWS credentials still work?

**Comment by tjsilver** [comment]
> LGTM.
> 
> Let's deploy to CODE before merging to ensure the changes to the AWS credentials still work?

I deployed to CODE, and ran the repocop lambda. It successfully ran and added two messages to the CODE fifo queue for branch protector.  I also ran the branch protector lambda, and it picked one message off the queue and didn't take any further action because it was CODE. So I think we're good to go.

**Pull request merged**